5.0/en/0x53-V53-WebRTC.md

```diff
@@ -47,6 +51,8 @@ Media security is paramount in WebRTC applications, as it directly impacts the c
 
 Signalling is a critical component of WebRTC applications, responsible for coordinating communication sessions between peers. The security of the signalling process is essential to prevent unauthorized access, eavesdropping, and service disruptions.
 
+These requirements only apply to systems that...
```

Comment:
@sandrogauci again, do you think we could add the circumstances here when this section will or will not be applicable?

Reply:
In this case, I cannot imagine realistic cases where you are not using some form of signalling server and also using your own WebRTC infrastructure. That said, the design is modular and we haven't seen everything yet. So I'll add basic text to be more explicit.

Reply:
Great, see comment above :)